### ISSUE TITLE
See thorugh between panels

### ISSUE THREAD

**Issue by maxwxyz** [obelisk79/OpenTheme#71]
The default background image is seen through gaps between panels which look unpleasant.
Maybe the icon could be removed at all.
![grafik](https://github.com/obelisk79/OpenTheme/assets/6246609/ec62b8a8-eb18-4996-8755-9f1a8c125e00)

**Comment by microbearsmb** [comment]
Yup, literally just checked this github to see if there was an issue for this. Looks weird to be able to see it between the panels, plus it is the older "sticker" version of Obelisk's logo. I love the idea of having a grayscale logo as the background when all tabs are closed though. Just needs to be updated to the new logo at some point.

**Comment by maxwxyz** [comment]
@obelisk79 shall I make an PR for changing the OpenTheme logo or removing the theme logo? If there is the need for a logo, it should not be visible between panels/views IMO.

**Comment by obelisk79** [comment]
I've updated the logo, however, even without that fact, I've been completely unable to reproduce this issue. If you can paste your about info that might help me find a way to at least recreate this effect.

**Comment by maxwxyz** [comment]
It's still visible in the latest weekly:
![image](https://github.com/obelisk79/OpenTheme/assets/6246609/2fb8142c-04ce-4d58-8740-764c0684e05b)
Just activate another panel and you get the gaps on both sides where the colors of the logo are visible.

```
OS: Windows 11 build 22631
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.37841 (Git)
Build type: Release
Branch: main
Hash: 5f761c758e2a0164b00a7183b9c0cbe32688e123
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: German/Germany (de_DE)
Installed mods: 
  * CfdOF 1.25.17
  * Curves 0.6.39
  * dodo 1.0.1
  * fasteners 0.5.24
  * freecad.gears 1.2.0
  * OpenTheme 2024.6.22
  * OpticsWorkbench 1.0.18
  * Rocket 3.3.0
  * sheetmetal 0.4.19

```

**Comment by obelisk79** [comment]
I am speculating this may be the result of either an undefined or transparent border, but the fun will be in narrowing down the offending widget type, there's a few which could be the culprit.

**Comment by obelisk79** [comment]
I'm just going to remove the image. It really offers nothing of tangible value.

**Comment by microbearsmb** [comment]
For 1.0, that sounds great. Beyond that though, if we can track down this bug, I think it's pretty neat and adds a nice touch of professionalism. Very minor aesthetic thing in the end though.
